mirage-block-ramdisk.opam

```diff
@@ -28,3 +28,4 @@ description: """
 - Can be dynamically resized
 - Supports querying sparseness information
 """
+x-maintenance-intent: [ "(latest)" ]
```

Comment:
I'm inclined to think it's `"(none)"`. The only users I could find are wodan and ocaml-qcow for tests, and neither has been updated for years and don't seem to see much usage.